src/TotkRstbGenerator.Core/RstbGenerator.cs

```diff
@@ -43,6 +43,32 @@ public RstbGenerator(string romfs, string? output = null, uint padding = 0)
         ArrayPool<byte>.Shared.Return(buffer);
     }
 
+    public RstbGenerator(string romfs, string? sourceRstbPath = null, string? output = null, uint padding = 0)
+    {
+        string path = sourceRstbPath ?? TotkConfig.Shared.RsizetablePath;
+        if (!File.Exists(path))
+        {
+            throw new FileNotFoundException($"The file '{sourceRstbPath}' could not be found.");
+        }
```

Comment:
`sourceRstbPath` should never be `null`

```suggestion
    public RstbGenerator(string romfs, string sourceRstbPath, string? output = null, uint padding = 0)
    {
        if (!File.Exists(sourceRstbPath)) {
            throw new FileNotFoundException($"The file '{sourceRstbPath}' could not be found.");
        }
```